examples/interaction/tools/position_toolbar.py

```diff
@@ -1,7 +1,18 @@
+''' A basic scatter plot with the toolbar positioned above the plot,
+outside of the axes, titles, etc.
+This example shows a way to set the toolbar's location relative to the plot,
+outside of the axes, titles, etc.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.toolbar_location, bokeh.plotting.figure.toolbar_sticky, bokeh.plotting.figure.scatter, bokeh.plotting.show
+    :refs: :ref:`_ug_interaction_tools_toolbar`
+    :keywords: position, location, sticky, toolbar
+'''
+
 from bokeh.plotting import figure, show
 
 p = figure(width=400, height=400,
-           title=None, toolbar_location="below",
+           title='Toolbar Location & Sticky Off', toolbar_location="above",
```

Comment:
Same comment here, the example used "below" for a reason. In the docs where this is embedded, it is specifically demonstrating how to achieve a location other than the default (which is already "above")

I would suggest that you confine the changes to the example to only adding the docstrings and metadata, and leaving the code as it currently is. If there's an idea to change something, you can open a separate issue to discuss.

Reply:
Got it, thank you @bryevdv. I'll revert the changes in the code.